docs/source/examples/links.rst

```diff
@@ -29,6 +29,13 @@ There are a few links you can use with different purposes.
           .. _here:
      - .. _here:
      - This is an internal cross reference bookmark. It requires an internal cross-reference anchor (above). It does not render, but serves as a point to link to.
+   * - Internal Cross-reference to Parameters
+     - .. code-block:: rst
+
+          :ref:`Link to a config param <confprop_parametr_name>`
+     - Renders the parameter name as a hyperlink.
+     - This is an internal cross-reference to a parameter on the `Configuration Parameters <https://opensource.docs.scylladb.com/stable/reference/configuration-parameters.html>`_ page.
```

Comment:
We might want to add that only works on the open-source & enterprise docs projects.